content-src/components/Spotlight/Spotlight.js

```diff
@@ -22,12 +23,23 @@ const SpotlightItem = React.createClass({
       bestImage: {}
     };
   },
+  onMouseIn(site) {
+    if (site.recommended) {
+      this.setState({hover: true});
+    }
+  },
+  onMouseOut(site) {
+    if (site.recommended) {
+      this.setState({hover: false});
+    }
+  },
   render() {
     const site = this.props;
     const image = site.bestImage;
     const imageUrl = image.url;
     const description = site.description || site.url;
     const isPortrait = image.height > image.width;
+    let timestamp = moment(site.timestamp).fromNow();
```

Comment:
this should be `const`, also can you name it something more descriptive like `relativeTimestamp`